authentication-service/build.gradle.kts

```diff
@@ -25,6 +24,7 @@ dependencies {
     implementation(projects.saveCloudCommon)
     implementation(libs.spring.boot.starter.security)
     implementation(libs.spring.security.core)
+    implementation("org.springframework:spring-jdbc")
```

Comment:
Did you tried to start some of applications, that used `auth-service` as dependency?
I remember, that I tried to use such solution sometime ago, but got errors.
Does it all okay now?

Reply:
Build was successful in that case, but nor the `backend` not `sandbox` didn't started

Reply:
Double checked, but I think for backend and sandbox it's safe - they have all spring-data related dependencies anyway. Also checked for gateway, which also has this dependency, and it's starting fine too.